test/jdk/jdk/crac/ContextOrderTest.java

```diff
@@ -51,21 +57,18 @@ public class ContextOrderTest {
     public static void main(String[] args) throws Exception {
         System.setProperty("java.util.logging.config.file", Utils.TEST_SRC + "/logging.properties");
 
-        testOrder();
-        testRegisterBlocks();
-        testThrowing();
-        testRegisterToCompleted();
-        testRegisterFromOtherThread();
+        Method m = ContextOrderTest.class.getDeclaredMethod(args[0]);
+        m.invoke(null);
     }
 
     private static void testOrder() throws Exception {
         var recorder = new LinkedList<String>();
         getGlobalContext().register(new MockResource(recorder, null, "regular1"));
-        getJDKContext().register(new MockResource(recorder, NORMAL, "jdk-normal"));
-        getJDKContext().register(new MockResource(recorder, SECURE_RANDOM, "jdk-later"));
-        getGlobalContext().register(new CreatingResource<>(recorder, null, "regular2", getJDKContext(), NORMAL));
+        JDKResource resource2 = new MockResource(recorder, NORMAL, "jdk-normal");
```

Comment:
Looks like the local vars are not used?

Reply:
Overlooked Resources are creating strong refs in their constructors.